### PR TITLE
Update bot.py

### PR DESCRIPTION
**Pull request by l-p-d**
added exception handling, idk how to properly make pull requests so there was probably an easier way

### CODE REVIEW

File: src/bot.py [hunk 1]
```diff
@@ -53,7 +53,25 @@ def int_conv(string: str) -> bool:
...
+       
+    # user gifting to != post author
+    elif comment.submission.author != args[1]:
+        comment.reply("ERROR_MESSAGE")
+        continue
+    
+    """
+    what needs to be done with the database:
+    elif int(args[2]) > author.vote_nugs:
+        comment.reply("ERROR MESSAGE")
```
Comment (xeoth) [inline src/bot.py:75]: So, the only person users will be able to give nuggets to is the post author?

Comment (l-p-d) [reply src/bot.py:75]: That's how I thought we wanted it, otherwise you could just give nuggets to a random friend/alt and it wouldn't make sense.

Comment (xeoth) [reply src/bot.py:75]: Good point. If that's the case, why not eliminate the second argument altogether?

Comment (l-p-d) [reply src/bot.py:75]: Ah true, I'd be all for that.

**Pull request merged**